hardware/index.md

```diff
@@ -219,15 +219,27 @@ Power to the micro:bit may be provided via 5V on the USB connector, or via a 3V
 
 The interface chip handles the USB connection, and is used for flashing new code to the micro:bit, sending and receiving serial data back and forth to your main computer.
 
+The V2.2X devices use two interface chip variants due to availability of the parts; 400K devices were built using nRF52833-QDAA and further devices are built using nRF52820-QDAA.
+
```

Comment:
As there are two different Interface chips with different specs it might be good to clarify here that having one is not better than the other, as the additional memory is not used.

```
As the two chip variants are of the same family, the same DAPLink firmware runs on either chip it and provides the exact same functionality.
```